test/test.js

```diff
@@ -135,7 +135,7 @@
   var isStrict = ui.isStrict;
 
   /** Used to test Web Workers. */
-  var Worker = !(ui.isForeign || ui.isSauceLabs || isModularize) && document && root.Worker;
+  var Worker = !(ui.isForeign || ui.isSauceLabs || isModularize) && document && document.origin != 'null' && root.Worker;
```

Comment:
Oh, Chome won't let you start a Worker if you're using a filepath as the URL. I can amend it out if you want.